src/main/java/muller/command/AddCommand.java

```diff
@@ -0,0 +1,121 @@
+package muller.command;
+
+import java.time.LocalDate;
+
+import muller.storage.Storage;
+import muller.task.Task;
+import muller.task.TaskList;
+import muller.ui.Ui;
+
+/**
+ * Command to add a task (Todo, Deadline, or Event) to the task list.
+ */
+public class AddCommand extends Command {
+    private String[] inputs;
```

Comment:
Perhaps the use of more descriptive names for method parameters would be better? eg. commandInputs or taskType